language/tutorial/tutorial.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START full_tutorial_script]
+# [START import_libraries]
+import argparse
+
+from googleapiclient import discovery
+
+from oauth2client.client import GoogleCredentials
+# [END import_libraries]
+
+
+def get_response(filename):
+    """Runs sentiment analysis on text within the specified file."""
+    # [START authenticating_to_the_api]
+    credentials = GoogleCredentials.get_application_default()
+    service = discovery.build('language', 'v1', credentials=credentials)
+    # [END authenticating_to_the_api]
+    # [START constructing_the_request]
+    with open(filename, 'r') as review_file:
```

Comment:
1) Put an empty newline above control statements
2) You didn't limit the scope of the with statement as I mentioned in the last round. :(